src/tools/wasm-ctor-eval.cpp

```diff
@@ -42,7 +43,7 @@ using namespace wasm;
 
 struct FailToEvalException {
   std::string why;
-  FailToEvalException(std::string why) : why(why) {}
+  FailToEvalException(std::string why) : why(std::move(why)) {}
```

Comment:
this move also puzzles me a little...?

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html